src/components/Profile/components/SchedulePanel/index.tsx

```diff
@@ -23,11 +23,15 @@ type Props = {
   myProf: boolean;
 };
 
+const scheduleOpenKey = 'profile.schedule.isOpen';
 const GordonSchedulePanel = ({ profile, myProf }: Props) => {
   const [loading, setLoading] = useState(true);
   const [allSchedules, setAllSchedules] = useState<Schedule[]>([]);
   const [selectedSchedule, setSelectedSchedule] = useState<Schedule | null>(null);
   const [selectedCourse, setSelectedCourse] = useState<CourseEvent | null>(null);
+  const [isScheduleOpen, setIsScheduleOpen] = useState<boolean>(
+    localStorage.getItem('scheduleOpenKey') === 'true' ?? true,
```

Comment:
This is using the **string** `'scheduleOpenKey'` as the key. What we want is to use the string `'profile.schedule.isOpen'` as the key, and the string is stored in the variable `scheduleOpenKey` (notice the lack of quotes).
```suggestion
    localStorage.getItem(scheduleOpenKey) === 'true' ?? true,
```